src/rohiniBDec19/RohiniAssignment3/Program8.java

```diff
@@ -0,0 +1,47 @@
+package rohiniBDec19.RohiniAssignment3;
+
+/*8) WAP to find vowels in given string  a. Input :- automation engineering b. Count :- 11 */
+
+class Program8 {
+	static int acount = 0;
+	static int ecount = 0;
+	static int icount = 0;
+	static int ocount = 0;
+	static int ucount = 0;
+
+	public static void main(String[] args) {
+		findVowels();
+		System.out.println(
+				"Count of vowels are a:" + acount + " e:" + ecount + " i:" + ecount + " o:" + ocount + " u:" + ucount);
+	}
+
+	private static void findVowels() {
+
+		String str = "automation engineering ";
+		for (int i = 0; i < str.length(); i++) {
+			switch (str.charAt(i)) {
+
+			case ('a' | 'A'):
```

Comment:
for 'A' it will not work.
make String in upper case or Lower case n then compare